.cirrus.yml

```diff
@@ -0,0 +1,46 @@
+freebsd_instance:
+  image_family: freebsd-12-1-snap
+
+test_task:
+  name: Tests (FreeBSD)
+  env:
+    matrix:
+      - PYTHON: python2.7
+      - PYTHON: python3.5
+      - PYTHON: python3.6
+      - PYTHON: python3.7
+      - PYTHON: python3.8
+  python_script:
+    - PYPACKAGE=$(printf '%s' $PYTHON | tr -d '.')
+    - SQLPACKAGE=$(printf '%s-sqlite3' $PYPACKAGE | sed 's/thon//')
+    - pkg install -y git-lite $PYPACKAGE $SQLPACKAGE
+  pip_script:
+    - $PYTHON -m ensurepip
+    - $PYTHON -m pip install pip -U
+    - $PYTHON -m pip install poetry -U --pre
+    - $PYTHON -m poetry config virtualenvs.in-project true
+  deps_script: $PYTHON -m poetry install
+  test_script: $PYTHON -m poetry run pytest -q --junitxml=junit.xml tests
```

Comment:
I can see we are uploading the `junit.xml` below, but I am assuming this is for the check comments? Can we make sure this only comments on errors?

Reply:
We can upload them only on failures if we don't want annotations for un-failed runs.